pyron-engine/src/test/scala/com/cloudentity/pyron/apigroup/ApiGroupConflictsSpec.scala

```diff
@@ -99,17 +99,13 @@ class ApiGroupConflictsSpec extends WordSpec with MustMatchers {
         GroupMatchCriteria(Some(BasePath("/x")), emptyDomains)
       ) mustBe true
       isConflicted(
-        GroupMatchCriteria(Some(BasePath("/x")), emptyDomains),
-        GroupMatchCriteria(None, emptyDomains)
+        GroupMatchCriteria(None, emptyDomains),
+        GroupMatchCriteria(Some(BasePath("/x")), emptyDomains)
       ) mustBe true
       isConflicted(
         GroupMatchCriteria(Some(BasePath("/x")), emptyDomains),
         GroupMatchCriteria(Some(BasePath("/x/y")), emptyDomains)
       ) mustBe true
-      isConflicted(
```

Comment:
Why is this case removed?

Reply:
please check my previous comment about allowing more specific rule not to conflict with more general rule that comes later